package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@material-ui/icons": "^1.0.0-beta.43",
     "@material-ui/lab": "^1.0.0-alpha.0",
+    "@skidding/react-codemirror": "^1.0.2",
```

Comment:
Switch to `react-codemirror2`. I commented with a usage of this I have done before.